packages/ui/src/views/connectors/connectors-list/connectors-list.tsx

```diff
@@ -111,12 +143,17 @@ export function ConnectorsList({
                     )}
                   </Button>
                 </Table.Cell>
-                <Table.Cell className="min-w-2 content-center !p-0 text-right">
+                <Table.Cell className="content-center !p-0">
                   <MoreActionsTooltip
                     actions={[
+                      {
+                        title: t('views:connectors.viewDetails', 'View Details'),
+                        onClick: () =>
+                          navigate(`${toConnectorDetails?.({ identifier, type, spec, status, lastModifiedAt })}`)
```

Comment:
let's use links instead of navigate functions

Reply:
+1